cache/remotecache/v1/utils.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	emptyGZLayer = digest.Digest("sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1")
```

Comment:
This const has been duplicated in several places: https://github.com/moby/buildkit/search?q=sha256%3A4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1

Can we use a single const?

Reply:
What package do you expect to define it? Currently, the packages that use it are all independent.

Reply:
I guess `exporter/containerimage/exptypes` maybe?

Reply:
exptypes sgtm

Reply:
done